### DISCUSSION TITLE
Consulta sobre proyecto

### DISCUSSION THREAD

**Discussion by LuciiAl** [harpomaxx/algoritmos2#77] [General]
Profes, queria consultar si van a poder subir los archivos de prueba en estos dias. 
Tambien tengo una duda particular, y es que hemos probado crear la base de datos con carpetas que contengan espacios en el nombre y no funciona, tampoco supimos arreglarlo, podrán darnos una mano con eso? o podemos tener en cuenta que no van a usar carpetas con esa caracteristica?

**Comment by harpomaxx** [comment]
ahora los estamos subiendo Lucia.
